.github/workflows/sbom_manifest_check.yml

```diff
@@ -0,0 +1,14 @@
+name: Regenerate and upload CycloneDX Manifest on changes to Cargo.lock files
+
+on:
+  push:
+    paths:
+      - Cargo.lock
+      - internal/shim-kvm/Cargo.lock
+      - internal/shim-sgx/Cargo.lock
+      - internal/wasmldr/Cargo.lock
+
+jobs:
+  Regenerate-BOM:
+    uses: abhinandanudupa/enarx/.github/workflows/sbom_manifest_action.yml@main
```

Comment:
This should be referring to the official Enarx repo rather than your own, right?

Reply:
Yes, I forgot this

Reply:
Only after the PR is accepted, the files be present  in the repo. So until then it will have to refer to my repo.

Reply:
Will have to tell the code owners about this.